src/MiddlewareInterface.php

```diff
@@ -8,15 +8,13 @@
 interface MiddlewareInterface
 {
     /**
-     * Process a client request and return a response.
-     *
-     * Takes the incoming request and optionally modifies it before delegating
-     * to the next frame to get a response.
+     * Process an incoming client or server request and return a response,
```

Comment:
This first line should be a short summary, followed by a longer description. As such, I think the original "title line" was correct, and this change should replace the description.